### DISCUSSION TITLE
Prevent static site from being found without having the link

### DISCUSSION THREAD

**Discussion by enricoschaaf** [vercel/next.js#14168] [Help]
I was wondering if there are any differences between getStaticProps and getServerProps when I don't want to site to be found without the user having the link. For example if you have a polling app and every poll has its own unique url and is static, does that make the poll easier to find compared to server rendering the unique url?

**Comment by jamesmosier** [comment]
By "prevent from being found" do you mean by being crawled by Google/other search engines?

If so, those would perform the same and be just as easily found no matter which you choose. At the end of the day the crawlers will render your site as it finds them, whether or not it is static or server.

**Reply by enricoschaaf** [reply]
No, I mean by malicious actors, I don't really know a lot about DNS etc. so I was wondering if it's more vulnerable because static files exist. For example when I generate a unique url for a user that I don't want to be found.
